.github/workflows/release.yml

```diff
@@ -156,3 +156,4 @@ jobs:
           body_path: ./builds/containerd-release-notes/release-notes.md
           files: |
             builds/release-tars-**/*
+          make_latest: false
```

Comment:
I think in main branch, we can set it as `true` . Even though any releases are not made from main, when a release branch is made from main, that will make the releases from new release branch automatically have the latest set to true.